flow/aggregates.py

```diff
@@ -30,10 +30,10 @@ def foo(*jobs):
         The default behavior is no sorting.
     :type sort_by:
         str or NoneType
-    :param reverse_order:
-        States if the jobs are to be sorted in reverse order.
-        The default value is False.
-    :type reverse_order:
+    :param sort_ascending:
```

Comment:
We should call this `sort_ascending` with a default value of `True` so it matches Pandas' `sort_values(by=..., ascending=True, ...)`. I find "ascending" unambiguous, while "reverse order" is less clear. https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.sort_values.html

Reply:
We sort in ascending order by default.
Hence we'll always have to use the `not` keyword.
While this will definitely work, my preference here would be using `sort_descending`

Reply:
I believe I only had to invert its value twice. Once in the call to `sorted` and also once in the check for “default values” where I replaced `not reverse_order` with `sort_ascending`. So it was an equal number of “nots” in each direction, which seems no more complex to me.

Reply:
I don't have a strong opinion either way, but this usage is distinct enough from anything you would do with pandas that I don't think there's any reason for confusion. I'm assuming the performance difference is negligible, but if that's not the case and the default sorting is the opposite of what pandas does I think choosing performance over exact API consistency with pandas is unnecessary here. I'm fine with whatever decision is made, though.

Reply:
@bdice @vyasr Then we can stick with how @bdice implemented.

Reply:
I don't feel this needs to conform to pandas, but I do think the naming is clearer with `sort_ascending` or `sort_descending`.